### PR TITLE
fix broken cobra command

### PR DESCRIPTION
**Pull request by DavidVorick**

### CODE REVIEW

**Comment by DavidVorick** [comment]
Merging immediately so I can get the other builds passing. Any comments/complaints can be handled retroactively.

File: siac/hostcmd.go [hunk 1]
```diff
@@ -358,7 +358,7 @@ func hostannouncecmd(cmd *cobra.Command, args []string) {
 	case 1:
 		err = post("/host/announce", "netaddress="+args[0])
 	default:
-		cmd.Usage()
+		cmd.UsageFunc()
```
Comment (lukechampine) [inline siac/hostcmd.go]: should be `c.UsageFunc()(c)`

Comment (DavidVorick) [inline siac/hostcmd.go]: What is the difference? I ran 'siac' to make sure that the usage output was correct

Comment (lukechampine) [inline siac/hostcmd.go]: are you sure you rebuilt the binary? `cmd.UsageFunc()` just returns a function, it doesn't print anything.

Comment (DavidVorick) [inline siac/hostcmd.go]: Oh, looks like I didn't rebuild the binary.

>.< derp

**Pull request merged**